packages/create-app/src/lib/tasks.ts

```diff
@@ -236,3 +236,59 @@ export async function moveAppTask(
       });
   });
 }
+
+/**
+ * Checks if git package is installed and git credentials exists
+ *
+ * @throws if `exec` fails
+ */
+export async function checkForGitSetup(): Promise<boolean> {
+  const runCmd = (cmd: string) =>
+    exec(cmd).catch(error => {
+      process.stdout.write(error.stderr);
+      process.stdout.write(error.stdout);
+      throw new Error(`Could not execute command ${chalk.cyan(cmd)}`);
+    });
+
+  try {
+    await runCmd('which git');
```

Comment:
Don't think this is gonna work on windows is it? We use `command-exists` for this in other places. I think we can add that dep, it's not too big